docs/source/swift-scripting.md

```diff
@@ -1,30 +1,29 @@
 ---
-title: Swift Scripting
+title: Swift scripting
+sidebar_title: Swift scripting (beta)
 ---
 
-⚠️ **PLEASE NOTE: THIS FUNCTIONALITY IS IN BETA** ⚠️
+> **This functionality is in beta.**
```

Comment:
I would prefer to keep the warning symbols here - the `>` renders this warning so lightly that it's easy to miss.